src/brsTypes/components/RoDeviceInfo.ts

```diff
@@ -0,0 +1,687 @@
+        },
+        impl: (_interpreter) => {
+            let result = new Array<AAMember>();
+            result.push({ name: new BrsString("playready"), value: new BrsString("tee;ss") });
+
+            return new RoAssociativeArray(result);
+        },
+    });
+
+    private getDrmInfoEx = new Callable("getDrmInfoEx", {
+        signature: {
+            args: [],
+            returns: ValueKind.Object,
+        },
+        impl: (_interpreter) => {
+            let result = new Array<AAMember>();
+            let subresult = new Array<AAMember>();
+            subresult.push({ name: new BrsString("multikey"), value: BrsBoolean.False });
+            subresult.push({ name: new BrsString("securestop"), value: BrsBoolean.True });
+            subresult.push({ name: new BrsString("tee"), value: BrsBoolean.False });
+            subresult.push({ name: new BrsString("version"), value: new Int32(2) });
+
+            result.push({
+                name: new BrsString("playready"),
+                value: new RoAssociativeArray(subresult),
+            });
+
+            return new RoAssociativeArray(result);
+        },
+    });
+
+    private getCaptionsMode = new Callable("getCaptionsMode", {
+        signature: {
+            args: [],
+            returns: ValueKind.String,
+        },
+        impl: (_interpreter) => {
+            return new BrsString("On");
+        },
+    });
+
+    private setCaptionsMode = new Callable("setCaptionsMode", {
+        signature: {
+            args: [new StdlibArgument("mode", ValueKind.String)],
+            returns: ValueKind.Boolean,
+        },
+        impl: (_interpreter) => {
+            return BrsBoolean.True;
+        },
+    });
+
+    private getCaptionsOption = new Callable("getCaptionsOption", {
+        signature: {
+            args: [new StdlibArgument("option", ValueKind.String)],
+            returns: ValueKind.String,
+        },
+        impl: (_interpreter) => {
+            return new BrsString("Default");
+        },
+    });
+
+    private getClockFormat = new Callable("getClockFormat", {
+        signature: {
+            args: [],
+            returns: ValueKind.String,
+        },
+        impl: (_interpreter) => {
+            return new BrsString("12h");
+        },
+    });
+
+    private enableAppFocusEvent = new Callable("enableAppFocusEvent", {
+        signature: {
+            args: [new StdlibArgument("enable", ValueKind.Boolean)],
+            returns: ValueKind.Dynamic,
+        },
+        impl: (_interpreter) => {
+            return new BrsString("Event notifications are enabled");
+        },
+    });
+
+    private enableScreensaverExitedEvent = new Callable("enableScreensaverExitedEvent", {
+        signature: {
+            args: [new StdlibArgument("enable", ValueKind.Boolean)],
+            returns: ValueKind.Dynamic,
+        },
+        impl: (_interpreter) => {
+            return new BrsString("Screen saver exit event notifications are enabled");
```

Comment:
Does this actually return a string? The BrightScript docs say that it returns a boolean reflecting whether or not the operation was a success.